test/general-use.js

```diff
@@ -33,17 +33,22 @@ module.exports = {
     english: {
 
         // english (appropriate mode) (sorted)
+        "Jack": "anga;quesse:a,tilde-below",
+        "happy": "hyarmen;parma:a,tilde-below;long-carrier:y",
+        "style": "silme;tinco;lambe:y,i-below",
+        "yellow": "anna;lambe:e,tilde-below;vala:o",
+        "phone": "formenparma;numen:o,i-below",
         "cake": "quesse;quesse:a,i-below",
         "cakes": "quesse;quesse:a;silme-nuquerna:e",
         "cats.": "quesse;tinco:a,s-final;full-stop", // regression
-        "green": "ungwe;romen;long-carrier:e;numen",
+        "green": "ungwe;romen;short-carrier:e;numen:e",
         "hobbits": "hyarmen;umbar:o,tilde-below;tinco:i,s-final",
         "hobbits'": "hyarmen;umbar:o,tilde-below;tinco:i,s-inverse",
         "hobbits''": "hyarmen;umbar:o,tilde-below;tinco:i,s-extended",
         "hobbits'''": "hyarmen;umbar:o,tilde-below;tinco:i,s-flourish",
-        "there": "thule;romen:e,i-below",
+        "there": "thule;ore:e,i-below",
```

Comment:
Can you restate concisely the rule for romen vs ore? My understanding is that they are both mode and language dependent, and may even in this case have special consideration given that the following vowel is silent. It would be good to include test cases to exercise every variation. Otherwise, we might end up whacking moles.

Reply:
When Tolkien used Rómen and Órë, he used them according to his dialect, which is an R-drop dialect. This means that the R was only retained when followed by a pronounced vowel ("there" would use Órë, but "therein" uses Rómen) but since the computer can't read pronunciation, I suggest if it is followed by a vowel (excluding silent E) that it automatically uses Rómen and uses Órë the rest of the time. This will get it close to being correct most of the time.

Reply:
Thanks again for the details. I agree we’d have a hard time capturing all the rules of the rhotic dialect.

These examples are good. Let’s capture all of them in cases, including the ones where we’d need to use backtick or full word exception to force the right output, like ```"ther`e`in"```.